index.js

```diff
@@ -705,6 +717,12 @@ const publicApi = {
         return this;
     },
 
+    configureResolveModules(directories) {
```

Comment:
Is this related to Aurelia? I mean, I know it's a webpack feature, but is it needed? I'm not familiar at all with Aurelia, so you'll have to help us out :)

Reply:
I'll give my best understanding, again, not super familiar with Webpack.

This is something that's required for Aurelia. Aurelia uses dynamic module loading a fair bit, which means that Webpack doesn't understand how to resolve those dynamic modules. This option then tells Webpack to try resolving via a particular path.

For example, my Webpack config code:

```
Encore
    // ...
    .configureResolveModules([
        './assets/app',
        'node_modules'
    ]);
```

You can read more about it on the aurelia/webpack-plugin [Wiki page](https://github.com/aurelia/webpack-plugin/wiki/Getting-started#whats-the-problem-with-aurelia-anyway).